src/main/java/nl/tudelft/ti2206/group9/level/State.java

```diff
@@ -65,7 +66,7 @@ public static void resetAll() {
         setCoins(0);
         highscore = 0;
         previousDistance = 0;
-        skin = Skin.getNoob();
+        skin = Skin.loadSkinsToList().get(0);
```

Comment:
Then at least make a constant out of it, something like `NOOB_SKIN_POS`. Even though it's not a "magic number", I agree it's still hardcoding a list position.
This has also been done with the position of the Player in Track's List<Entity>.

<review,logic